tests/runtest.cmd

```diff
@@ -73,11 +75,13 @@ if /i "%1" == "longgc"                (set __LongGCTests=1&shift&goto Arg_Loop)
 if /i "%1" == "gcsimulator"           (set __GCSimulatorTests=1&shift&goto Arg_Loop)
 if /i "%1" == "jitstress"             (set COMPlus_JitStress=%2&shift&shift&goto Arg_Loop)
 if /i "%1" == "jitstressregs"         (set COMPlus_JitStressRegs=%2&shift&shift&goto Arg_Loop)
-if /i "%1" == "jitminopts"            (set COMPlus_JITMinOpts=1&shift&goto Arg_Loop)
+if /i "%1" == "jitminopts"            (set COMPlus_JitMinOpts=1&shift&goto Arg_Loop)
 if /i "%1" == "jitforcerelocs"        (set COMPlus_ForceRelocs=1&shift&goto Arg_Loop)
 if /i "%1" == "jitdisasm"             (set __JitDisasm=1&shift&goto Arg_Loop)
 if /i "%1" == "ilasmroundtrip"        (set __IlasmRoundTrip=1&shift&goto Arg_Loop)
 if /i "%1" == "GenerateLayoutOnly"    (set __GenerateLayoutOnly=1&shift&goto Arg_Loop)
+if /i "%1" == "SkipGenerateLayout"    (set __SkipGenerateLayout=1&shift&goto Arg_Loop)
+if /i "%1" == "BuildXUnitWrappers"    (set __BuildXunitWrappers=1&shift&goto Arg_Loop)
```

Comment:
nit: casing of this `set __Build...` is inconsistent with the original definition casing.